cedar-drt/fuzz/src/schemas.rs

```diff
@@ -189,6 +189,12 @@ impl<K: Eq + Ord + Display, V: Equiv> Equiv for BTreeMap<K, V> {
     }
 }
 
+impl<N: Clone + PartialEq + Debug + Display + TypeName + Ord> Equiv for json_schema::CommonType<N> {
+    fn equiv(lhs: &Self, rhs: &Self) -> Result<(), String> {
+        Equiv::equiv(&lhs.ty, &rhs.ty)
+    }
```

Comment:
should we check more than just the `.ty` fields?  is the other field annotations, should we check that the annotations are the same?

Reply:
We'll do it later: https://github.com/cedar-policy/cedar-spec/pull/492#discussion_r1887196692